### DISCUSSION TITLE
Cannot chain commands beyond `omz update` if update is available

### DISCUSSION THREAD

**Discussion by mikefrancis** [ohmyzsh/ohmyzsh#12821] [Help] [answered]
This is a very minor annoyance that I can certainly live with but asking just in case someone has the answer.

I have a cron scheduled that runs `omz update && brew update` every week. This works well, unless OhMyZsh actually has an update. If this is the case, the update is applied but `brew update` never seems to happen as the OhMyZsh update hard exits.

I'm just wondering if it's possible to modify this command to not hard exit after `omz update`?

**Comment by carlosala** [comment, marked as answer]
You can use `omz update --unattended` for that 👍🏻

**Reply by mikefrancis** [reply]
Thanks so much!
